config-sets/config03/revert-config.sh

```diff
@@ -0,0 +1,80 @@
+#include the path of the IS Pack for the productHome variable when trying locally
+#productHome=$productHome
+productHome=/home/kavitha/PRODUCTS/QA_Hackathon/APRIL/wso2is-5.5.0
+
```

Comment:
Shall we remove the manually given path and keep the product home as a variable to export

Reply:
@ShanikaWickramasinghe , Please see the fix in commit 144b21c43720b06551628ec64e380c6034cd9195